### DISCUSSION TITLE
.lat domains not supported

### DISCUSSION THREAD

**Discussion by AbrahamX3** [vercel/community#4371] [Feature Question] [answered]
### Question

I recently bought a new .lat domain and tried to transfer it to Vercel but got the following error:

![image](https://github.com/vercel/community/assets/78459953/2fff4407-117a-4d18-9eba-57d6ebd8f767)

Are .lat domains going to be supported soon in the future or will I have to purchase a new different domain?

**Comment by amyegan** [comment, marked as answer]
Hi @AbrahamX3! We don't support transferring or buying `.lat` domains right now, but you can still use a `.lat` domain with your Vercel project if you own it through a different registrar. You can use `A`/`CNAME` records or change the nameservers to point your custom domain at your Vercel project. 

You can find a walkthrough for adding a custom domain at https://vercel.com/docs/projects/domains/add-a-domain
